packages/tools/pie-icons-vue/package.json

```diff
@@ -46,7 +46,9 @@
     "lint:scripts:fix": "yarn lint:scripts --fix",
     "prebuild:example": "yarn build:icons",
     "prepublishOnly": "yarn lint:scripts && yarn build",
-    "test": "run -T vitest run"
+    "test": "run -T vitest run",
+    "test:ci": "yarn test",
```

Comment:
Have you considered not making scripts specific to CI?

Most tools already make use of the environment variable `CI=true` to trigger CI specific behaviour. That includes `vitest`.

> _**Note:** The `CI` variable is always set in mostly every modern CI system._

I've found that usually it's quite nice to in your workflow use exactly the same config as the test people run but with additional flags. For example:

```yaml
  - name: Tests running from CI
    run: yarn test --coverage
```

I'd further argue that to streamline the `scripts` section you'd ideally have as few scripts as possible (which ends up still amounting to quite a few) that are specifically meant to be run by developers (only). And then augmenting them in CI as mentioned above.

Reply:
This is a great point and something we're looking to address soon. At the moment though we should really get the tests running to make sure they don't get broken. Unfortunately we were too late to save a few packages from that fate 😢